app/assets/stylesheets/basic.scss

```diff
@@ -1,5 +1,5 @@
 #background {
-  background: url("www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
+  background: url("http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
```

Comment:
URL `http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png` should not contain protocol or domain